src/qt/utilitydialog.cpp

```diff
@@ -58,7 +58,8 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
         ui->helpMessage->setVisible(false);
     } else {
         setWindowTitle(tr("Command-line options"));
-        QString header = "Usage:  bitcoin-qt [command-line options]                     \n";
+        QString header = "Usage:  bitcoin-qt [command-line options] [BIP21 URI]                     \n\n"
```

Comment:
What is the purpose of these extra spaces on the end? Do they need to be adjusted? Can't seem to figure it out from the git history either :/

Reply:
I think it can be removed, it doesn't make any difference. I couldn't find any other reference in the code of such a thing (extra spaces at the end of a line before an \n). I think it was some formatting logic left behind.

Regarding its historical origins, it seems that it was coming from the `bitcoind`/ `bitcoin-cli` helps and copied over ([#13872](https://github.com/bitcoin/bitcoin/pull/13872)).
You can still see it at some places (exactly the same 22 spaces):
![image](https://github.com/bitcoin-core/gui/assets/110166421/0fbe1cf6-2d6b-4690-b1f2-e393f5bda0ec)

Reply:
Also, I'd remove 1 space after "Usage:", so the start of "bitcoind" matches the alignment of the options helps.
```suggestion
        QString header = "Usage: bitcoin-qt [command-line options] [BIP21 URI]                     \n\n"
```

Reply:
I agree with @luke-jr and @pablomartin4btc , I ll remove the extra spaces at the end, I see no reason to leave them. Regarding the help2man tool mentioned in https://github.com/bitcoin/bitcoin/pull/13872 , there is no reason for extra spaces.